react/Autosuggest/Autosuggest.js

```diff
@@ -122,10 +124,10 @@ export default class Autosuggest extends Component {
     };
 
     const allInputProps = {
+      ...omit(this.props, [ 'inputProps', 'labelProps', 'autosuggestProps' ]),
       ref: this.storeTextFieldReference,
       inputProps: enrichedInputProps,
-      labelProps: enrichedlabelProps,
-      ...omit(this.props, [ 'inputProps', 'autosuggestProps' ])
```

Comment:
No reason, just dragged it...that was causing some overwriting before I added 'lableProps' that's why it ended up in another place